src/OrchardCore/OrchardCore.Queries.Abstractions/ElasticQueryResults.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace OrchardCore.Queries
+{
+    public class ElasticQueryResults : IQueryResults
```

Comment:
I'm not sure this is necessary as maybe we should rename the LuceneQueryResults class to SearchEngineQueryResults and reuse it.

Reply:
Done